### ISSUE TITLE
Call to undefined method [package]

### ISSUE THREAD

**Issue by dragonfire1119** [dwightwatson/sitemap#9]
There's a error with the ``SitemapServiceProvider.php on line 34``

```
/**
	 * Bootstrap the application events.
	 *
	 * @return void
	 */
	public function boot()
	{
		//$this->package('watson/sitemap');
	}
```

**Comment by dwightwatson** [comment]
Yeah, this is due to a change in Laravel 5 just before it was released. There is a PR (#8) to fix this.

**Comment by dragonfire1119** [comment]
For now I just created my own facade until you get time to fix it for Laravel 5. Thanks for your time and work on this thanks.

**Comment by dwightwatson** [comment]
The PR has now been merged and tagged `2.0.3`, so you should be able to use that without issue now.

**Comment by dragonfire1119** [comment]
@dwightwatson Thanks when I get sometime ill try it out.
